### PR TITLE
Release Roslyn Project System and Nugets which are Release versioned

### PR DESCRIPTION
**Pull request by basoundr**
**Customer scenario**

Nugets that we produce today are built using the Pre-release version of some of the dependencies. This change will remove those Pre-release version dependencies and hence will enable us to release release-versioned Nuget packages for Roslyn Project system. The binaries that get built and packaged into the Nuget will be available to insert into VS

**Bugs this fixes:**

[388822](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/388822)

**Workarounds, if any**

No. All VS nugets are non-pre release so it makes sense to release non-pre release nugets for Project system as well

**Risk**

There has not been much code churn with the dependencies and we are taking a dependency over VS release quality package. So we hope the risk is minimal. We will do manual validation in addition to this.

**Performance impact**

none

**Is this a regression from a previous update?**

No

**Root cause analysis:**

CPS just released their Nuget with non-pre-release version. Without their package, this work is not possible

**How was the bug found?**

We have been tracking this for sometime now.

Tagging @natidea @srivatsn @davkean @dotnet/project-system for review

### CODE REVIEW

**Comment by basoundr** [comment]
Output from my test branch can be seen `\\cpvsbuild\Drops\Roslyn\Roslyn-Project-System\Roslyn-Project-System-Signed\20170227.3\NuGetPackages`

File: build/Targets/VSL.Versions.targets [hunk 1]
```diff
@@ -61,14 +61,13 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <VSSemanticVersion>15.0.1</VSSemanticVersion>
+    <VSSemanticVersion>15.0</VSSemanticVersion>
```
Comment (srivatsn) [inline build/Targets/VSL.Versions.targets]: We're going back a version? Isn't this going to break upgrades in VS?

Comment (mavasani) [inline build/Targets/VSL.Versions.targets]: Yes, this will break upgrades, please revert this.

Comment (basoundr) [inline build/Targets/VSL.Versions.targets]: Nuget versions has 3 parts. `Major`.`Minor`.`Patch`. Later, the property is updated to append the BuildNumber and the Number of build for the day. Example version would look like this `15.0.6142703`. Look at the above location where I have the sample output.

Comment (srivatsn) [inline build/Targets/VSL.Versions.targets]: But I think this variable affects the generated VSIX version as well.

**Comment by srivatsn** [comment]
Tagging @MattGertz

File: build/Targets/VSL.Versions.targets (hunk 1 above)
Comment (basoundr) [inline build/Targets/VSL.Versions.targets]: The name makes it sound generic. But `VSSemanticVersion` is used just for Nuget version and not outside.

Comment (basoundr) [inline build/Targets/VSL.Versions.targets]: The name makes it sound generic. But VSSemanticVersion is used just for Nuget version and not outside.

Comment (srivatsn) [inline build/Targets/VSL.Versions.targets]: Ok looks like this variable was bumped up unnecessarily before. I verified from the drop share that the vsix version is still 15.0.1.xxxx and that comes from https://github.com/dotnet/roslyn-project-system/pull/1639/files#diff-ce3a3697b9bd06fb295da18808269b37R3

Comment (mavasani) [inline build/Targets/VSL.Versions.targets]: Shouldn't we still keep the VSIX version in sync with assembly version?

Comment (mavasani) [inline build/Targets/VSL.Versions.targets]: Sorry, I meant shouldn't we keep the NuGet version same as VSIX and assembly versions?

Comment (mavasani) [inline build/Targets/VSL.Versions.targets]: Okay, I understand it now. Ignore my comments, 👍

**Comment by natidea** [comment]
Approved in shiproom

**Pull request merged**